doc/convert.md

```diff
@@ -47,14 +73,14 @@ UYVY10bit LE (1 pixel group)
 ```
 
 ### rfc4175_422be10
-Color space: YUV  
```

Comment:
Please keep the last space of each line as end-of-line in md

Reply:
Yes. All note goes to one line if you see https://github.com/corco/Media-Transport-Library/blob/updated_docs/doc/convert.md#rfc4175_422le10

Reply:
Fixed